src/table/components/__tests__/TableBodyWrapper.spec.jsx

```diff
@@ -12,30 +12,28 @@ import * as selectionsUtils from '../../utils/selections-utils';
 import * as getCellRenderer from '../renderer';
 import * as handleKeyPress from '../../utils/handle-key-press';
 import * as handleAccessibility from '../../utils/handle-accessibility';
+import RootContext from '../../../contexts/rootContext';
 
 describe('<TableBodyWrapper />', async () => {
   const model = { getHyperCubeData: async () => generateDataPages(2, 2) };
-
-  let tableData;
-  let constraints;
-  let selectionsAPI;
-  let layout;
-  let theme;
+  let providerProps;
   let cellRendererSpy;
 
   beforeEach(async () => {
     sinon.stub(selectionsUtils, 'addSelectionListeners').returns(sinon.spy());
-
-    tableData = await manageData(model, generateLayout(1, 1), { top: 0, height: 100 });
-    constraints = {};
-    selectionsAPI = {
-      isModal: () => true,
-    };
-    theme = {
-      getColorPickerColor: () => {},
-      name: () => {},
+    providerProps = {
```

Comment:
i think a test util function is required here in order to get some arguments (based on the test case that we are writing it) and gives us whatever react-testing-library apis, or something similar to that. 

the purpose should be only prevent  providing the context in all test cases using `<RootContext.Provider value={...}>`